proto/c1/connector/v2/resource.proto

```
@@ -4,6 +4,8 @@
 
 import "google/protobuf/any.proto";
 import "google/protobuf/struct.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
```

Buf (1.47.2)

Import "google/protobuf/timestamp.proto" is unused.

(IMPORT_USED)